scrapinghub/client.py

```diff
@@ -1018,3 +1138,22 @@ def get(self, key, *args, **kwargs):
         if key is None:
             raise ValueError("key cannot be None")
         return self._origin.get(key, *args, **kwargs)
+
+    def set(self, *args, **kwargs):
+        """Set item to collection by key.
+
+        The method returns None (original method returns an empty generator).
+        """
+        self._origin.set(*args, **kwargs)
+
+    def delete(self, _keys):
```

Comment:
Renamed (initially was done for consistency with original method, but it's a not an argument)